compiler/test/dotty/tools/utils.scala

```diff
@@ -58,7 +59,8 @@ def toolArgsParse(lines: List[String]): List[String] = {
   val endc = "*" + "/"    // be forgiving of /* scalac: ... */
   def stripped(s: String) = s.substring(s.indexOf(tag) + tag.length).stripSuffix(endc)
   val args = lines.to(LazyList).take(10).filter { s =>
-       s.contains("// " + tag)
+       s.contains("//" + tag)
+    || s.contains("// " + tag)
     || s.contains("/* " + tag)
```

Comment:
Should these also be accepted?
* `"//  " + tag` with 2 or more spaces or a tab
* `"/*" + tag` with no spaces

Reply:
Maybe we should consider a regex

Reply:
Less (ways) is more, IMO.  I wouldn't have even added "/* ", but someone did back in scala/scala so I kept it.

Reply:
They're called s'mores for a reason. I see partest doesn't look at opening comment. Could filter on tag and then on open comment preceding it. As sample UX, I learned yesterday that commenting out an `// error` line does not suppress the error expectation. So temporarily commenting out requires two edits. My takeaway is that a test can be forgiving, but should help disallow "my check file was not used" or "my options were not used" and I didn't know it and the test has been broken all along.